shell/src/main/java/alluxio/cli/ValidateConf.java

```diff
@@ -31,15 +31,15 @@
    * @param args there are no arguments needed
    */
   public static void main(String[] args) {
-    int ret = 0;
     LOG.info("Validating configuration.");
-    if (Configuration.validate()) {
-      LOG.info("All configuration entries are valid.");
-    } else {
-      LOG.info("Configuration has invalid entries.");
-      ret = -1;
+    try {
+      Configuration.validate();
+      LOG.info("Configuration is valid.");
+    } catch (IllegalStateException e) {
+      LOG.warn("Configuration is invalid: {}", e.getMessage());
```

Comment:
Log this at error level? I don't think we should put the stack trace, but the end result is crashing the system which seems to be more of an issue than warn.